src/utility.ts

```diff
@@ -129,3 +129,14 @@ export function getFirefoxArgs(
     "about:blank",
   ];
 }
+
+//This methods just waits for the browser endpoint
+export async function waiter() {
+  await delay(1000); //Waiting normally
+  try {
+    await (await fetch("http://localhost:9292/json/version")).body?.cancel();
+  } catch (_ex) {
+    const endTime = (new Date()).getTime() + 3000; //waiting abnormally xD
+    while ((new Date()).getTime() < endTime);
+  }
```

Comment:
```suggestion
  try {
    await (await fetch("http://localhost:9292/json/version")).body?.cancel();
  } catch (_ex) {
    await waiter()
  }
```

Reply:
I actually tried this, but the recursion was causing async ops issue, I will try it once again though, just to be sure